Skype/Trusted-Application-API/SDK/Tests/ClientModel/AudioVideoFlow.cs

```diff
@@ -197,6 +211,33 @@ public void StatePropertyShouldGetUpdatedOnEvent()
         }
 
         [TestMethod]
+        public async Task StopPromptsShouldReturnOnlyOnPromptCompletedEvent()
+        {
+            // Given
+            TestHelper.RaiseEventsFromFile(m_mockEventChannel, "Event_AudioVideoFlowConnected.json");
+            TaskCompletionSource<bool> requestReceived = new TaskCompletionSource<bool>();
+            m_restfulClient.HandleRequestProcessed += (sender, args) =>
+            {
+                if (args.Uri == new Uri(DataUrls.StopPrompts) && args.Method == HttpMethod.Post)
+                {
+                    requestReceived.SetResult(true);
+                }
+            };
+
+            // When
+            Task promptTask = m_audioVideoFlow.PlayPromptAsync(new Uri("https://example.com/prompt"), m_loggingContext);
```

Comment:
you want to wait for the Task returned by StopPromptAsync(); not by PlayPromptAsync(). I mean promptTask should be assigned the return value of StopPromptAsync()